api/src/main/java/ca/bc/gov/educ/api/gradstudent/service/DataConversionService.java

```diff
@@ -350,7 +351,8 @@ private void populate(OngoingUpdateFieldDTO field, GraduationStudentRecordEntity
     private String getStringValue(Object value) {
         if (value == null)
             return null;
-        return (String) value;
+        String str = (String) value;
```

Comment:
Need safety checking to avoid a classCastException. Maybe use if(value instanceof String){...}

Reply:
The update is committed.